api/gobgp.proto

```diff
@@ -125,7 +125,7 @@ message WatchEventRequest {
 
   message Table {
     message Filter {
-      enum Type { BEST = 0; ADJIN = 1; POST_POLICY = 2; }
+      enum Type { BEST = 0; ADJIN = 1; POST_POLICY = 2; EOR = 3; }
```

Comment:
Is this necessary? ADJIN is supposed to send all messages include EoR?

Reply:
> Is this necessary? ADJIN is supposed to send all messages include EoR?

Adding a new filter provides opportunity of getting eor message itself without handling adj-in tables that can be huge.
I thought about adding it to Best actually, but it will probably not the right place (tell me if I'm wrong).

Reply:
I see. Is this backward compatible?

Reply:
It looks so. If you don't add a filter, you won't get an eor message.

Reply:
The old binary (using Watch gRPC API) works with the gobgpd binary including this change?